packages/wdio-cli/src/launcher.js

```diff
@@ -173,6 +173,13 @@ class Launcher {
              * fail if no specs were found or specified
              */
             if (Object.values(this.schedule).reduce((specCnt, schedule) => specCnt + schedule.specs.length, 0) === 0) {
+                for (let s in this.configParser.getConfig().exclude) {
```

Comment:
You should be able to grab these values once instead of loading the config for each value:
```js
const { specs, exclude } = this.configParser.getConfig();
```

Another question though, can we grab the specs and exclude options from the cap obj passes to runMode?